var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -173,7 +173,7 @@ class Trilinos(CMakePackage):
     conflicts('+superlu-dist', when='+superlu')
     # For Trilinos v11 we need to force SuperLUDist=OFF, since only the
     # deprecated SuperLUDist v3.3 together with an Amesos patch is working.
-    conflicts('+superlu-dist', when='@:11.14.3')
```

Comment:
oh, actually i know why. That's because of the ordering of versions. `xdsk-0.2.0` is split into `xsdk`, `0`, `2`, `0`. When compared to `11`, `14`, `3` you have `xsdk` < `11` thus this condition is satisfied. What you want to do is add a lower bound. Just use the lowest normal trilinos version in Spack -- `11.14.1`.